src/cswinrt/code_writers.h

```diff
@@ -1427,7 +1427,9 @@ remove => %.% -= value;
                 }
             }
             // Skip metadata attributes
-            if (attribute_namespace == "Windows.Foundation.Metadata" && attribute_name != "DefaultOverload" && attribute_name != "Overload") continue;
+            if (attribute_namespace == "Windows.Foundation.Metadata" && 
+                attribute_name != "DefaultOverload" && attribute_name != "Overload" && attribute_name != "AttributeUsage") 
+                continue;
             attributes[attribute_full] = std::move(params);
         }
         if (auto&& usage = attributes.find("AttributeUsage"); usage != attributes.end())
```

Comment:
Any concerns with the below not finding anything given that would probably get skipped:

            bool allow_multiple = attributes.find("Windows.Foundation.Metadata.AllowMultiple") != attributes.end();

Reply:
yup - fix coming for that